src/pages/filter/ui/panels/function-panel/components/compound-het/compound-het.store.ts

```diff
@@ -18,38 +17,35 @@ export const CompoundHetSelectOptions = [
 
 class CompoundHetStore {
   statFuncStatus = ''
+  initialApprox: string | null = ''
 
   currentMode?: ModeTypes
 
   constructor() {
     makeAutoObservable(this)
   }
 
-  public setCurrentMode(modeType: ModeTypes): void {
-    this.currentMode = modeType
+  public setInitialApprox(initialApprox: string | null) {
+    this.initialApprox = initialApprox
   }
 
-  public resetCurrentMode(): void {
-    this.currentMode = undefined
+  public resetInitialApprox() {
+    this.initialApprox = ''
   }
 
-  public get cachedValues(): ICompoundHetCachedValues {
-    return functionPanelStore.getCachedValues<ICompoundHetCachedValues>(
-      FuncStepTypesEnum.CompoundHet,
-    )
+  public setCurrentMode(modeType?: ModeTypes): void {
+    this.currentMode = modeType ?? undefined
```

Comment:
modeType parameter already either ModeTypes or undefined, why specify that one more time?